src/main/java/network/brightspots/rcv/ContestConfig.java

```diff
@@ -515,6 +515,26 @@ private void validateOutputSettings() {
       validationErrors.add(ValidationError.OUTPUT_CONTEST_NAME_MISSING);
       Logger.severe("contestName is required!");
     }
+
+    if (!SecurityConfig.canOutputFilesSaveToUsersDirectory()) {
+      try {
+        String homeDirectory = new File(System.getProperty("user.home")).getCanonicalPath();
+        // Strip the username off the file path to get the root Users directory path
+        String rootUsersDirectory = homeDirectory.substring(0, homeDirectory.lastIndexOf('\\'));
+        String outputDirectory = new File(getOutputDirectory()).getCanonicalPath();
+        if (outputDirectory.startsWith(rootUsersDirectory)) {
```

Comment:
I'm struggling with the entire premise of this PR -- please see https://github.com/BrightSpots/rcv/issues/757#issuecomment-1793502225.

Reply:
Let me know if your concerns were addressed there

Reply:
Yep. It's a lot to test, so  I'm just going to assume this will solve the issue as expected.